pkgs/top-level/python-packages.nix

```diff
@@ -18164,7 +18164,7 @@ in modules // {
   };
 
 
-  setuptoolsTrial = buildPythonPackage {
+  setuptoolsTrial = makeOverridable ({ twisted ? self.twisted }: buildPythonPackage {
     name = "setuptools-trial-0.5.12";
```

Comment:
could you add a comment here explaining why you use makeOverridable and for which package this is important.